ice/trace.py

```diff
@@ -41,7 +41,7 @@ def make_id() -> str:
 def _url_prefix():
     if codespace := os.environ.get("CODESPACE_NAME"):
         return f"https://{codespace}-3000.githubpreview.dev"
```

Comment:
I assume this 3000 is also a port? Not sure what to do about this, I'm guessing the codespace is already broken and I don't know what the plan for it is.